java/src/main/java/ai/rapids/cudf/ColumnViewPointerAccess.java

```diff
@@ -0,0 +1,45 @@
+/*
+ *
+ *  Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package ai.rapids.cudf;
+
+//Evolving and Unstable
+
+/**
+ * Interface that allows access only methods to cudf column_views
+ */
+interface ColumnViewPointerAccess<T> {
```

Comment:
In the spark plugin we need a way to get the child types, or else we are going to have to change a massive amount of code.  Can we please make this public?